search-commons/src/test/java/no/unit/nva/search/models/SearchResponseDtoTest.java

```diff
@@ -38,6 +49,24 @@ void jsonSerializationShouldKeepDeprecatedFieldsUntilFrontendHasMigrated() throw
         assertThat(json, is(jsonObject().where("total", JsonMatchers.jsonLong(searchResponse.getSize()))));
     }
 
+    @Test
+    void searchResponseShouldMapSnakeCaseToCamelCase() throws IOException {
+        var searchResponse = getSearchResponse();
+        var aggregations = SearchResponseDto.fromSearchResponse(searchResponse, randomUri()).getAggregations();
+
+        var docCount = aggregations.findValue("docCount");
+        Assertions.assertNotNull(docCount);
+        Assertions.assertEquals(2, docCount.asInt());
+
+        var snakeCase = aggregations.findValue("doc_cound_error_upper_bound");
+        assertThat(snakeCase, is(nullValue()));
+    }
+
```

Comment:
As I understand it your code only renames a few hardcoded values, and you test that theese are renamed here, but can you also add a test that makes sure we dont rename anything we are not supposed to?